BSP/STM32F413/Src/BSP_SPI.c

```diff
@@ -1,8 +1,28 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
 #include "BSP_SPI.h"
 #include "stm32f4xx.h"
+#include "os.h"
+#include "BSP_OS.h"
+
+static bsp_os_t *os;
+static bsp_os_t *os3;
```

Comment:
I was trying to avoid things like this when I added the LUTs below. Let's avoid putting a whole bunch of `if`s everywhere if possible.